docs/op-guide/deploy-tikv-using-docker-compose.md

```diff
@@ -20,63 +20,143 @@ Make sure you have installed the following items on your machine:
     sudo yum install docker docker-compose
     ```
 
-- Helm
-
-    ```bash
-    curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
-    ```
-
 - Git
 
     ```
     sudo yum install git
     ```
 
-## Install and deploy
+## Install 
 
-1. Download `tidb-docker-compose`.
+Download `tidb-docker-compose`.
 
-    ```bash
-    git clone https://github.com/pingcap/tidb-docker-compose.git
+```bash
+git clone https://github.com/pingcap/tidb-docker-compose.git
+```
+
+## Prepare cluster
+
+Here we will run a simple cluster with only 1 PD and 1 TiKV. The base docker compose config is:
```

Comment:
In this example, let's run a simple cluster with only 1 PD server and 1 TiKV server. See the following for the basic docker compose configuration file: